src/cpr_data_access/models.py

```diff
@@ -1110,6 +1110,12 @@ def to_huggingface(
             info=dataset_info,
         )
 
+        rename_map = {
+            "languages": "document_languages",
+        }
+
+        huggingface_dataset = huggingface_dataset.rename_columns(rename_map)
+
```

Comment:
What's the reason for the renaming here?

Reply:
it's to avoid confusion with the text block language 'language' field if directly using our open data. i've added a comment to explain this.